tools/iptb-plugins/filecoin/local/localfilecoin.go

```diff
@@ -129,27 +129,15 @@ func init() {
 	}
 }
 
+// We can't copy the bits in go due to the following bug
+// https://github.com/golang/go/issues/22315
 func copyFileContents(src, dst string) (err error) {
-	in, err := os.Open(src)
-	if err != nil {
-		return
-	}
-	defer in.Close() // nolint: errcheck
-	out, err := os.Create(dst)
-	if err != nil {
-		return
-	}
-	defer func() {
-		cerr := out.Close()
-		if err == nil {
-			err = cerr
-		}
-	}()
-	if _, err = io.Copy(out, in); err != nil {
-		return
+	cmd := exec.Command("cp", src, dst)
```

Comment:
This won't work in windows. At the very least we should at a compiler flag to note that.

Reply:
The original bug probably doesn't exist in windows, so I can probably move this out to a new file and flag it.